build/tasks/task-serve.js

```diff
@@ -25,8 +26,14 @@ gulp.task('serve', async () => {
     '--port', ports[0],
   ];
 
+  const runnerArgs = [
+    ...args,
+    ...(yargs.argv.browser ? ['--browser', yargs.argv.browser] : []),
```

Comment:
I'm not sure if this browser arg is needed since opn it will use your default browser otherwise.  Are you wanting it to open in a different browser?